### PR TITLE
[JBIDE-20370] Display the OpenShift Explorer when the 'New Application' wizard completes

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

**Comment by jcantrill** [comment]
@fbricon LGTM.  Merge away....or is that something @adietish or I need to do?

**Comment by fbricon** [comment]
I have god powers over all JBT repos, I can do it :-)

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/NewApplicationWizard.java [hunk 1]
```diff
@@ -43,7 +45,9 @@
  * @author Andre Dietisheim
  */
 public class NewApplicationWizard extends Wizard implements IWorkbenchWizard, IConnectionAwareWizard<Connection> {
-	
+
+	private static final String OPENSHIFT_EXPLORER_VIEW_ID = "org.jboss.tools.openshift.express.ui.explorer.expressConsoleView";
+
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/NewApplicationWizard.java]: This very much looks like a constant that we should allow others to use. and thus should extract to some public class

Comment (fbricon) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/NewApplicationWizard.java]: Indeed, but I had no other use case and didn't where else to put it. In the spirit of YAGNI, I left it here :-)

**Pull request merged**